### ISSUE TITLE
COS integration not working properly

### ISSUE THREAD

**Issue by marowid** [canonical/bundle-kubeflow#531]
Steps to reproduce: 
install COS Lite - juju deploy ./cos-lite-bundle.yaml --trust --overlay ./offers-overlay.yaml
juju add-relation jupyter-controller:grafana-dashboard admin/cos.grafana-dashboards

After integration you can see Grafana Dashboards with following issues:
- in general lack of any guide on how to integrate and use observability with Kubeflow
- be default nothing is visible (showing only an empty COS dashboard)
- after setting up proper filters you can see data that don't reflect reality. Ie. this is a view from Kubeflow having 2 notebooks running (it shows 5 and 1 of something - as the labels don't tell what it actually is)
<img width="771" alt="Zrzut ekranu 2022-12-14 o 17 04 37" src="https://user-images.githubusercontent.com/11966308/207646481-85bf1d9b-40f4-4f8f-a0f0-2e134b25c8fb.png">

This bug is a sample for jupyter notebooks but same behavior can be observed for other components as well.

**Comment by i-chvets** [comment]
Low priority.

**Comment by andreeamun** [comment]
I think the purpose was to drop the dashboards until we figure it out the COS integration itself. I highly doubt that any team will accept the handover without visibility on at least some metrics regarding the tool. @i-chvets .

@marowid , please disagree if I am wrong

**Comment by marowid** [comment]
yes, we cannot just drop it compleatly. We need at least basic observability to call it a product

**Comment by i-chvets** [comment]
Seldon dashboard fix: https://github.com/canonical/seldon-core-operator/pull/96
- Old dashboard was referencing non-existing Seldon metrics.
- No explanation of what is executor API and how it should be used.
- Create simple dashboard that shows how many models are deployed/rejected.

**Comment by i-chvets** [comment]
Argo workflow dashboard is verified to show correct information.

**Comment by i-chvets** [comment]
Jupyter notebook controller dashboard fix: https://github.com/canonical/notebook-operators/pull/203
- Updated labels to reflect metrics displayed.
- Updated graphs.

**Comment by i-chvets** [comment]
Kubeflow COS integration documentation is updated here: https://github.com/canonical/kubeflow-examples/pull/11

**Comment by i-chvets** [comment]
To test above fixes refer to Kubeflow COS Integration guide.
[README.md](https://github.com/canonical/kubeflow-examples/blob/kf-833-doc-kubeflow-cos-integration/kubeflow-cos-integration/README.md)

**Comment by i-chvets** [comment]
All fixes have been merged.
